. LD_LIBRARY_PATH is useless in robotpkg. And better not set it anyway.

--- cmake/templates/_setup_util.py.in~	2018-06-06 18:18:22.000000000 +0200
+++ cmake/templates/_setup_util.py.in	2018-09-04 14:56:08.672323000 +0200
@@ -59,7 +59,6 @@
 # subfolder of workspace prepended to CMAKE_PREFIX_PATH
 ENV_VAR_SUBFOLDERS = {
     'CMAKE_PREFIX_PATH': '',
-    'LD_LIBRARY_PATH' if not IS_DARWIN else 'DYLD_LIBRARY_PATH': @CATKIN_LIB_ENVIRONMENT_PATHS@,
     'PATH': PATH_TO_ADD_SUFFIX,
     'PKG_CONFIG_PATH': @CATKIN_PKGCONFIG_ENVIRONMENT_PATHS@,
     'PYTHONPATH': '@PYTHON_INSTALL_DIR@',
